tools/bin/hawqpylib/HAWQ_HELP.py

```diff
@@ -159,10 +159,10 @@
 
 The "options" are:
    -c --change         Changes a configuration parameter setting.
```

Comment:
We'd better not change it, HAWQ have many client option names from GPDB, sometimes it's good to have similar options for the users who use both. And if we change any options, we have to change it together with Ambari.